namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-production/resources/rds.tf

```diff
@@ -0,0 +1,24 @@
+module "rds-instance" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.0"
```

Comment:
Can you use the latest version please `4.5`
for guidance
`https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/blob/master/example/rds.tf`

Reply:
thanks @vijay-veeranki-moj - have updated the version